src/sentry/api/endpoints/organization_events.py

```diff
@@ -160,21 +161,13 @@ def data_fn(offset, limit):
             )
         except discover.InvalidSearchQuery as error:
             raise ParseError(detail=six.text_type(error))
-        except (snuba.SnubaError, snuba.QueryOutsideRetentionError) as error:
-            logger.info(
-                "organization.events.snuba-error",
```

Comment:
I've removed this logging as we haven't been checking it and we'll be capturing the important stuff in sentry going forwards.